### ISSUE TITLE
Define iOS and Android Accessibility conventions

### ISSUE THREAD

**Issue by shalomhalbert** [2xe-icp/planning-icp#5]
To help developers contribute, we should have two documents that highlight accessibility best practices for building an accessible application for visually, hearing and physically impaired/disabled users.
